netlify.toml

```
@@ -6,3 +6,23 @@
   from = "/"
   to = "/business-admin"
   status = 302
+
+[[redirects]]
+  from = "/business-admin/quickstart/paymentmethod.html"
+  to = "/business-admin/billing/payment-method.html"
+  status = 302
+
+[[redirects]]
+  from = "/business-admin/account/contact.html"
+  to = "/business-admin/billing/invoice-contact.html"
+  status = 302
+
+[[redirects]]
+  from = "/business-admin/aes/aessetup.html"
+  to = "/business-admin/aes/setup.html"
+  status = 302
+
+[[redirects]]
+  from = "/business-admin/aes/aesproduct.html"
+  to = "/business-admin/aes/setup.html"
```

```suggestion
  to = "/business-admin/aes/product.html"
```
I think this should be `product.html` and not `setup.html`. Or we should add a comment mentioning that this is on purpose.

---

No, this is on purpose. I have combined both articles in one -> /aes/setup. Admin guide should not duplicate the content on our website. This is an old article written by Casey.